src/app/views/panels/scale_editor.tsx

```diff
@@ -89,12 +89,11 @@ export class ScaleEditor extends React.Component<
                     text={strings.scaleEditor.add}
                     onClick={() => {
                       const mappingsKey = Object.keys(scale.properties.mapping);
-                      const theLastMapping: any = mappingsKey[
-                        mappingsKey.length - 1
-                      ] as any;
+                      const theLastMapping: string =
+                        mappingsKey[mappingsKey.length - 1];
                       const value = (scale.properties.mapping as any)[
```

Comment:
Also another `any` here, perhaps you can cast to `unknown` and then to the type you want.

Reply:
Cast to AttributeMap